functions/api/image/nfts/asset/[[index]].tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable import/no-unused-modules */
+import { ImageResponse } from '@vercel/og'
+import React from 'react'
+
+import getAsset from '../../../../utils/getAsset'
```

Comment:
non relative if possible

Reply:
i don't know if i can but will try

Reply:
I don't think you can in this case. I'm not sure what the base url would be, and wrangler wraps this up with its own opaque webpack config.